ts4.1/index.d.ts

```diff
@@ -0,0 +1,265 @@
+import { PascalCase, CamelCase } from "type-fest";
+
+declare namespace camelcaseKeys {
+	/**
+	 * @internal
+	 * Dispatches to PascalCase or CamelCase.
+	 *
+	 * @param S a union of string literal types to camelcase.
+	 * @param UsePascalCase if true, the first word is also capitalized.
+	 */
+	type Camelize<S extends string, UsePascalCase extends boolean = false> =
```

Comment:
I don't think these helper types need to be in the namespace. They can just be top-level in this file.